### PR TITLE
Topic/xlsx download fix

### PR DESCRIPTION
**Pull request by darekbienkowski**
Description 
-----------------------------------------------------

Fix for downloading xlsx files from breeders/search/wizard page.

Two changes for input select values from xls to xlsx. 

Checklist 
---------------------------------------------------------------------------------------
- [ ] Refactoring only
- [ ] Documentation only
- [ ] Fixture update only
- [x] Bug fix
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] New feature
  - [ ] Relevant tests have been created and run.
  - [ ] Data was added to the fixture
    - [ ] Data was added via a patch in `/t/data/fixture/patches/`.
  - [ ] User-Facing Change
    - [ ] The user manual in `/docs` has been updated.
  - [ ] Any new Perl has been documented using **perldoc**.
  - [ ] Any new JavaScript has been documented using **JSDoc**.
  - [ ] Any new _legacy_ JavaScript has been moved from `/js` to `/js/source/legacy`.

### CODE REVIEW

File: mason/breeders_toolbox/program/upload_profile_dialog.mas [hunk 1]
```diff
@@ -133,7 +133,7 @@ $program_name
                         Product Profile Template
                     </b>
                     <ul>
-                        <li>Product profile template is an Excel file (.xls format)</li>
+                        <li>Product profile template is an Excel file (.xls)</li>
```
[the remark this answers is not in the document]
Comment (lukasmueller) [reply mason/breeders_toolbox/program/upload_profile_dialog.mas:136]: Hi Bryan, probably, but this pull request is for an urgent fix, so we should deal with this one later.

**Review by bellerbrock — APPROVED** [review]

File: mason/breeders_toolbox/program/upload_profile_dialog.mas (hunk 1 above)
[the remark this answers is not in the document]
Comment (darekbienkowski) [reply mason/breeders_toolbox/program/upload_profile_dialog.mas:136]: The profile template has some own problems but is not connected with xlsx files... 
I tested it on test casava, and there is a download problem as on dockers... 
I wasn't sure if it a configuration of a path to 

`static/documents/tempfiles/other/excelXXXX`

https://github.com/solgenomics/sgn/blob/efba125c10375e4ad6f3996672fed691faef472f/lib/SGN/Controller/AJAX/BreedingProgram.pm#L539

or some other problem with that controller, that is why I left it as is.

[the remark this answers is not in the document]
Comment (darekbienkowski) [reply mason/breeders_toolbox/program/upload_profile_dialog.mas:136]: Screenshot from 2022-12-13 22-38-02

[the remark this answers is not in the document]
Comment (bellerbrock) [reply mason/breeders_toolbox/program/upload_profile_dialog.mas:136]: Oh weird, yeah must be a configuration issue. The download works for me on my development machine, but also fails for me too on cassava-test.

[the remark this answers is not in the document]
Comment (darekbienkowski) [reply mason/breeders_toolbox/program/upload_profile_dialog.mas:136]: We can check later where is a problem with the configuration, and then I will change the instruction, file format and extension to xlsx, but for a moment, that was for me out of scope for that fix.

**Review by lukasmueller — APPROVED** [review]
:-)

**Pull request merged**